Outreachy Dec 2021 to Mar 2022.md

```diff
@@ -0,0 +1,36 @@
+Oppia is excited to be participating on [Outreachy](https://www.outreachy.org/) December 2021 to March 2022 round! Outreachy is a diversity initiative that provides paid, remote internships to people subject to systemic bias and impacted by underrepresentation in the technical industry where they are living.
+
+Thank you for your interest in contributing with our project! Here you can find initial guidance on how to get started with the required tasks listed on the project [Scale Community Outreach and Conduct Research on Oppia's Lessons](https://www.outreachy.org/outreachy-december-2021-internship-round/communities/oppia/#scale-community-outreach-and-conduct-research-on-o).
+
+If you have any questions, feel free to send them to outreachy-questions@oppia.org or send them **directly to mentors by email**.
+
+## Getting Started
+
+Please follow these steps to get started:
+
+### User Studies
+
+*  For the user studies, start reading [this guide](https://docs.google.com/document/d/1j1S9HnoesGZMNeYccV8rXZ2bZwPhKV5-B6j9CnPRM_I/edit?usp=sharing)  that provides information on how to properly perform a user study/learner feedback session on Oppia's math lessons. You can also use [this checklist](https://docs.google.com/spreadsheets/d/1ybGCKSsa8fIqcBliavShN06f_PakiiBPPxLg6d48R2w/edit?usp=sharing) to help you don't miss any important steps. 
```

Comment:
Do you expect the contributors to make a copy of the google sheet? If so can you mention that?

Reply:
Just added that!